api/database/migrations/2019_11_09_123934_rename_column_for_job_forms_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class RenameColumnForJobFormsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('job_forms', function(Blueprint $table) {
```

Comment:
I would've prefered to have both tables changed in one migration but this works too.

Reply:
no fiex needed